Resources/public/templates/fields/edit/relation.hbt

```diff
@@ -0,0 +1,36 @@
+<div class="pure-g ez-editfield-row">
+    <div class="pure-u ez-editfield-infos">
+        {{> ez_fieldinfo_tooltip }}
+        <label for="ez-field-{{ content.contentId }}-{{ fieldDefinition.identifier }}">
+            <p class="ez-fielddefinition-name">
+                {{ fieldDefinition.names.[eng-GB] }}{{#if isRequired}}*{{/if}}:
+            </p>
+            <p class="ez-editfield-error-message">&nbsp;</p>
+        </label>
+    </div>
+    <div class="pure-u ez-editfield-input-area ez-default-error-style">
+        <ul class="ez-relation-values">
+            {{#if isEmpty}}
+               <li>"This field is empty"</li>
+                <button class="ez-discover ez-button font-icon pure-button">Select a relation</button>
+            {{else}}
+                {{#if destinationContent }}
+                     <li>"{{ destinationContent.name }}"</li>
+                     <li>Published: "{{ destinationContent.publishedDate }}"</li>
+                     <li>Last modification date: "{{ destinationContent.lastModificationDate }}"</li>
+                     <button class="ez-discover ez-button font-icon pure-button">Select a relation</button>
```

Comment:
your markup is invalid. A `ul` element (`ez-relation-values`) can only have `li` elements as children, while here you generate some buttons and a paragraph below.